package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
- [ ] I can't see the `react-dom`, and `babel-loader` packages in your package.json file kindly make sure to install the necessary packages so the app works as expected